### DISCUSSION TITLE
pressure sensitivity on freehand sketches (tablet support)

### DISCUSSION THREAD

**Discussion by Lik-the-Fluffin** [jgraph/drawio#4580] [General]
* [X ] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [X ] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Is your feature request related to a problem? Please describe.**
I use draw.io very often as I need to communicate ideas with colleagues on a regular basis. Draw.io works so nice so that. I often use its integration into the obsidian framework as it allows me to quickly make notes along my diagrams and connect them for future reference. Since I'm a visual person I tend to make quick sketches that support my notes and I do that with a tablet. The workflow to do integrations into notes is sometimes a pain and sketches get lost regularly. Draw.io already has a freehand tool, so an all-in-one solution would benefit my overall workflow for creating diagrams and sketches. It only misses pressure sensitivity which is a must have for good looking, accurate sketches.

**Describe the solution you'd like**
I'd like to improve the current freehand tool with:

   - pressure sensitivity when using a pen tablet (e.g. wacom)
   - quick access to related tools, "freehand drawn" lines, circles, ..
   - an eraser
   - a color picker
   - quick access to settings of brush tool (size, Flow, Smoothing, ...)

If Im not mistaken theres an open source freehand library that could be integrated:
https://github.com/steveruizok/perfect-freehand

and a demo of it in action:
https://perfect-freehand-example.vercel.app/

**Describe alternatives you've considered**
As a workaround for my flawed workflow (obsidian for notes handling, draw.io plugin for inline diagrams), I use the Obsidian Excalidraw plugin. However it lack a lot of required tools and it doesn't allow for inline sketches like draw.io does. I very much would like a similar functionality in draw.io (concerning sketches) since it already has such a great selection of tools. Freehand pressure sensitivity would really complete the toolset of draw.io.

**Additional context**
https://github.com/jgraph/drawio/issues/3141#issue-1431116914 Was auto closed by stale bot

**Comment by davidjgraph** [comment]
Thanks for the suggestion, but this way too overloaded to be a single feature request, so I'll move it to a discussion. The freehand library you list is indeed the one we use.

**Reply by bjornasm** [reply]
Would it be possible to open up for different pen types, eraser etc? It would be great to do free hand annotations and drawing alongside the current capabilities.
